spec/operators/finally-spec.ts

```diff
@@ -71,4 +74,90 @@ describe('Observable.prototype.finally', () => {
       .share()
       .subscribe();
   });
+
+  it('should handle empty', () => {
+    let executed = false;
+    let s1 = hot('|');
+    let result = s1.finally(() => executed = true);
+    let expected = '|';
+    expectObservable(result).toBe(expected);
+    // manually flush so `finally()` has chance to execute before the test is over.
+    rxTestScheduler.flush();
+    expect(executed).to.equal(true);
+  });
+
+  it('should handle never', () => {
+    let executed = false;
+    let s1 = hot('-');
+    let result = s1.finally(() => executed = true);
+    let expected = '-';
+    expectObservable(result).toBe(expected);
+    // manually flush so `finally()` has chance to execute before the test is over.
+    rxTestScheduler.flush();
+    expect(executed).to.equal(false);
+  });
+
+  it('should handle throw', () => {
+    let executed = false;
+    let s1 = hot('#');
+    let result = s1.finally(() => executed = true);
+    let expected = '#';
+    expectObservable(result).toBe(expected);
+    // manually flush so `finally()` has chance to execute before the test is over.
+    rxTestScheduler.flush();
+    expect(executed).to.equal(true);
+  });
+
+  it('should handle basic hot observable', () => {
+    let executed = false;
+    let s1 = hot(  '--a--b--c--|');
+    let subs =     '^          !';
+    let expected = '--a--b--c--|';
+    let result = s1.finally(() => executed = true);
+    expectObservable(result).toBe(expected);
+    expectSubscriptions(s1.subscriptions).toBe(subs);
+    // manually flush so `finally()` has chance to execute before the test is over.
+    rxTestScheduler.flush();
+    expect(executed).to.equal(true);
+  });
+
+  it('should handle basic cold observable', () => {
+    let executed = false;
+    let s1 = cold(  '--a--b--c--|');
+    let subs =      '^          !';
+    let expected =  '--a--b--c--|';
+    let result = s1.finally(() => executed = true);
+    expectObservable(result).toBe(expected);
+    expectSubscriptions(s1.subscriptions).toBe(subs);
+    // manually flush so `finally()` has chance to execute before the test is over.
+    rxTestScheduler.flush();
+    expect(executed).to.equal(true);
+  });
+
+  it('should handle basic error', () => {
+    let executed = false;
+    let s1 = hot(  '--a--b--c--#');
+    let subs =     '^          !';
+    let expected = '--a--b--c--#';
+    let result = s1.finally(() => executed = true);
+    expectObservable(result).toBe(expected);
+    expectSubscriptions(s1.subscriptions).toBe(subs);
+    // manually flush so `finally()` has chance to execute before the test is over.
+    rxTestScheduler.flush();
+    expect(executed).to.equal(true);
```

Comment:
just note : now true / false assertion can be written as `expect(something).to.be.true`